### ISSUE TITLE
Check "run at a glance" coloring

### ISSUE THREAD

**Issue by jaycedowell** [lwa-project/session_schedules#9]
@ctaylor-physics says that all of the lines are the same color.

**Comment by jaycedowell** [comment]
@ctaylor-physics can you give 4e922f4 a try?

**Comment by ctaylor-physics** [comment]
Looks like there is a least recently used cache file that is MIA on this version of LSL. Maybe this is a leftover from other commits and isn't relevant anymore but here is the traceback:

Traceback (most recent call last):
  File "/home/ctaylor/session_schedules/swarmGUI.py", line 38, in <module>
    from calibratorSearch import CalibratorSearch as OCS
  File "/home/ctaylor/session_schedules/calibratorSearch.py", line 29, in <module>
    from lsl.misc.lru_cache import lru_cache
ModuleNotFoundError: No module named 'lsl.misc.lru_cache'

**Comment by jaycedowell** [comment]
Yeah, that module's been removed in the yet-to-be-released LSL 3.

**Comment by jaycedowell** [comment]
What about now?

**Comment by ctaylor-physics** [comment]
I think this looks good for the telescope color codes. The legend always gets a bit wonky for more conventional interferometry runs with multiple sources. We could add indications for color-telescope pairs but I'm not sure how to balance that with the target scans other than pinning telescopes at the top of the legend. I've included some images to illustrate what I mean.

**Single Sources:**
![swarmRunAtGlance](https://github.com/user-attachments/assets/0baf615d-3ae6-48ab-b3b9-a0a849051498)

**Lots of sources from a "more conventional run":**
 I also noticed that the length of the legend from a run like this tends to mess with the plot shape. For example, this looks nice without messing with the plot at all, but if I zoom in on one of the scans the plot becomes smooshed trying to fit the whole legend inside the figure bounds as it re-calculates the aspect scaling. Doesn't impact functionality, but food for thought. I can make a figure showing this behavior if you like too. 

![swarmRAGmany](https://github.com/user-attachments/assets/d9fc928c-27b0-46de-889c-32684c12649e)
